pkgs/servers/home-assistant/custom-components/huawei_solar/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  huawei-solar,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "wlcrs";
+  domain = "huawei_solar";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    inherit owner;
```

```suggestion
  owner = "wlcrs";
  domain = "huawei_solar";
  version = "1.4.1";

  src = fetchFromGitHub {
    owner = "wlcrs";
```
Those strings just match by coincidence